src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -2632,7 +2632,7 @@ def create_express_route_peering(
         cmd, client, resource_group_name, circuit_name, peering_type, peer_asn, vlan_id,
         primary_peer_address_prefix, secondary_peer_address_prefix, shared_key=None,
         advertised_public_prefixes=None, customer_asn=None, routing_registry_name=None,
-        route_filter=None, legacy_mode=None):
+        route_filter=None, legacy_mode=None, ip_version='IPv4'):
```

Comment:
Where does 'ip_version' argument come from?

Reply:
It is described in SDK and could also be found in _params.py